vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -53,25 +53,37 @@ function onDoubleClick(e){
 }
     
 function uploadImage() {
-    var file = document.getElementById('file').files[0];
+    
+    var file = $('#file')[0].files[0]
     var reader  = new FileReader();
     var formData = new FormData();
     formData.append('file', file);
     formData.append('contentOrder', contentCount);
-    if ($(".open")[0]){
-        var imageBlockId = $('.open img').attr('id')
+   
+     
+    // Ashmi changes for Story VSPC-64
+    
+    if ($(".imgDiv").attr('id') && contentCount != 0){
```

Comment:
I'm confused, how could that work? there are potentially multiple image elements that have the class imgDiv.